app/process/legacy_surfaceQuality/surfaceQuality.lua

```diff
@@ -62,22 +61,13 @@ function osm2pgsql.process_way(object)
     return
   end
 
-  local surface, surface_source = SurfaceDirect(tags.surface)
+  local surface, surface_source = DeriveSurface(tags)
 
   -- Try to find smoothess information in the following order:
   -- 1. `smoothess` tag
   -- 2. `smoothess` extrapolated from surface data
```

Comment:
Move to `DeriveSurface`

Reply:
There is already a more detailed version of this comment. I deleted this one from the legacy file